wiki/Featured_Artists/es.md

```diff
@@ -0,0 +1,11 @@
+# Artistas destacados
+
+**¡Los artistas destacados** son artistas musicales que han trabajado con osu! para hacer una selección de sus pistas disponibles para uso sin restricciones en la comunidad. ¡Algunos de los artistas destacados también han compuesto pistas específicamente para osu! y los solté como osu! exclusivas
+
+[Una sección del sitio web](https://osu.ppy.sh/beatmaps/artists) enumera y detalla cada artista destacado. En las páginas de los artistas, puede encontrar información general sobre ellos, así como todas las pistas con licencia para su uso en osu. acompañado de archivos `.osz` pre-programados para su uso en beatmapping.
+
+## Gremio de mapeador'es
+
+Para promover el mapeo del contenido de los artistas destacados, los miembros de la comunidad crearon el **Mapper's Guild**, dándoles a los mapeadores un sistema de búsqueda de estilo MMO en el que pueden participar con otros mapeadores. ¡Las recompensas por ser activo en el gremio incluyen insignias de perfil, acceso temprano al nuevo contenido de artista destacado y la capacidad de ayudar a decidir qué artistas osu! el equipo debería considerar trabajar con el próximo.
```

Comment:
the title is `Mappers' Guild` with the apostrophe after `s` (as in, a guild for all mappers) everywhere in this article

should probably write that instead of `Gremio de mapeador'es` too.

Reply:
Fixed

Reply:
`Mapper's Guild` => `Mappers' Guild`

Reply:
Fixed